test-framework-docker/scheduler/src/main/java/com/containersolutions/mesoshelloworld/scheduler/FrameworkScheduler.java

```diff
@@ -33,8 +44,8 @@ public void resourceOffers(SchedulerDriver driver, List<Offer> offers) {
             if (currentOffer.isAcceptable() && (acceptedOffers < MAX_OFFERS)) {
 
                 System.out.println(
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS106)